security/cmd/node_agent/na/nodeagent.go

```diff
@@ -158,12 +158,12 @@ func (na *nodeAgentInternal) createRequest() ([]byte, *pb.Request, error) {
 		RSAKeySize: na.config.RSAKeySize,
 	})
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to generate CSR: %v", err)
```

Comment:
The prefix here is to differentiate the two error messages. When debug, simply look at the 'err' here is hard to know which line actually fails, it can be this line or line 166.

Now since the prefixes are different, you can easily know which line fails when debugging.

I don't mind rewording them, like capitalize the first letter or something.